### PR TITLE
Added css classes depending on popover state

### PR DESCRIPTION
**Pull request by bsekachev**
### Motivation and context
To solve some issues during testing we need additional css classes, that appear after any popover animations. 

### How has this been tested?
Manual testing

### Checklist

- [x] I submit my changes into the `develop` branch
- [ ] I have added description of my changes into [CHANGELOG](https://github.com/opencv/cvat/blob/develop/CHANGELOG.md) file
- [ ] I have updated the [documentation](
  https://github.com/opencv/cvat/blob/develop/README.md#documentation) accordingly
- [ ] I have added tests to cover my changes
- [ ] I have linked related issues ([read github docs](
  https://help.github.com/en/github/managing-your-work-on-github/linking-a-pull-request-to-an-issue#linking-a-pull-request-to-an-issue-using-a-keyword))
- [ ] I have increased versions of npm packages if it is necessary ([cvat-canvas](https://github.com/opencv/cvat/tree/develop/cvat-canvas#versioning),
[cvat-core](https://github.com/opencv/cvat/tree/develop/cvat-core#versioning), [cvat-data](https://github.com/opencv/cvat/tree/develop/cvat-data#versioning) and [cvat-ui](https://github.com/opencv/cvat/tree/develop/cvat-ui#versioning))

### License

- [x] I submit _my code changes_ under the same [MIT License](
  https://github.com/opencv/cvat/blob/develop/LICENSE) that covers the project.
  Feel free to contact the maintainers if that's a concern.
- [x] I have updated the license header for each file (see an example below)

```python
# Copyright (C) 2020 Intel Corporation
#
# SPDX-License-Identifier: MIT
```

### CODE REVIEW

**Review by dvkruchinin — CHANGES_REQUESTED** [review]

**Comment by dvkruchinin** [comment]
I also noticed a change in the popover view for tag 
setup_tag_popover

File: tests/cypress/support/commands.js [hunk 1]
```diff
@@ -154,7 +154,7 @@ Cypress.Commands.add('switchLabel', (labelName, objectType) => {
     const pattern = `^(Draw new|Setup) ${objectType}$`;
     const regex = new RegExp(pattern, 'g');
     cy.contains(regex)
-        .parents('.cvat-draw-shape-popover-content')
+        .parents(objectType === 'tag' ? '.cvat-setup-tag-popover-content' : '.cvat-draw-shape-popover-content')
```
[the remark this answers is not in the document]
Comment (bsekachev) [reply tests/cypress/support/commands.js]: Sure, I suppose we do not need using regex in this case.

**Comment by bsekachev** [comment]
> I also noticed a change in the popover view for tag
> setup_tag_popover

Fixed

File: tests/cypress/support/commands.js (hunk 1 above)
[the remark this answers is not in the document]
Comment (dvkruchinin) [reply tests/cypress/support/commands.js]: Thanks. Yes, now that there is a separation of popovers for the type of object being created, there is no need for using regex.

**Pull request merged**